### ISSUE TITLE
Cannot use js file to launch next start (12.1.5 and later)

### ISSUE THREAD

**Issue by luxirbt** [vercel/next.js#36893]
### Verify canary release

- [x] I verified that the issue exists in Next.js canary release

### Provide environment information

    Operating System:
      Platform: linux
      Arch: x64
      Version: #1 SMP Debian 4.19.208-1 (2021-09-29)
    Binaries:
      Node: 16.13.1
      npm: 8.2.0
      Yarn: 1.22.17
      pnpm: N/A
    Relevant packages:
      next: 12.1.6
      react: 18.0.0
      react-dom: 18.0.0

### How are you deploying your application? (if relevant)

next start

### Describe the Bug

When running npm run start, we got : 
```shell
ready - started server on 0.0.0.0:3014, url: http://localhost:3014
warn  - Port 3000 is in use, trying 3001 instead.
warn  - Port 3001 is in use, trying 3002 instead.
warn  - Port 3002 is in use, trying 3003 instead.
ready - started server on 0.0.0.0:3003, url: http://localhost:3003
Error: Could not find a production build in the '/home/xxx/xxx/frontend/.next' directory. Try building your app with 'next build' before starting the production server. https://nextjs.org/docs/messages/production-start-no-build-id
    at NextNodeServer.getBuildId (/home/xxx/xxx/frontend/node_modules/next/dist/server/next-server.js:137:23)
    at new Server (/home/testapp/xxx/frontend/node_modules/next/dist/server/base-server.js:93:29)
    at new NextNodeServer (/home/xxx/xxx/frontend/node_modules/next/dist/server/next-server.js:86:9)
    at NextServer.createServer (/home/xxx/xxxx/frontend/node_modules/next/dist/server/next.js:109:16)
    at async /home/xxx/xxx/frontend/node_modules/next/dist/server/next.js:121:31
```

### Expected Behavior

```sh
ready - started server on 0.0.0.0:3014, url: http://localhost:3014
```

### To Reproduce

Create a file prod-server.js with : 
```js
const cli = require("next/dist/cli/next-start");
cli.nextStart(["-p", "3014"]);
```

In package.json :
`"start": "node prod-server.js"`

**Comment by balazsorban44** [comment]
This kind of usage is undocumented, and thus there are also no semver guarantees.

Could you add a reproduction and also explain what you are trying to achieve?

**Comment by balazsorban44** [comment]
This issue has been automatically closed because it received no activity for a month and had no reproduction to investigate. If you think this was closed by accident, please leave a comment. If you are running into a similar issue, please open a new issue with a reproduction. Thank you.
